src/components/AddressSearch/index.tsx

```diff
@@ -243,15 +136,15 @@ function AddressSearch({
             country: countryFallbackLongName = '',
             state: stateAutoCompleteFallback = '',
             city: cityAutocompleteFallback = '',
-        } = GooglePlacesUtils.getPlaceAutocompleteTerms(autocompleteData.terms);
+        } = GooglePlacesUtils.getPlaceAutocompleteTerms(autocompleteData?.terms ?? []);
 
-        const countryFallback = _.findKey(CONST.ALL_COUNTRIES, (country) => country === countryFallbackLongName);
+        const countryFallback = Object.keys(CONST.ALL_COUNTRIES).find((country) => country === countryFallbackLongName);
 
-        const country = countryPrimary || countryFallback;
+        const country = countryPrimary ?? countryFallback ?? '';
```

Comment:
Is it possible that `countryPrimary` will be `''`?

Reply:
Right, it might be